modules/ROOT/pages/to-deprecate-asset.adoc

```diff
@@ -6,8 +6,8 @@ endif::[]
 Deprecating removes access to a version of an asset. The asset remains accessible for users.
 A deprecated asset can be reverted and returned to use.
 
-You must be an organization owner or an Exchange Administrator to use this feature. You can set these roles
-in the Access Management > Roles menu.
+You must be an asset administrator, an organization owner or an Exchange Administrator to use
```

Comment:
should "Administrator" for Exchange be upper case vs "asset administrator," where it's lower

Reply:
It should be "administrator". Fixed. Thank you!